autovalet_gazebo/launch/valet_rtab_sim.launch

```diff
@@ -47,7 +47,7 @@
         <!-- ICP parameters -->
         <param name="Icp/PointToPlane"        type="string" value="true"/>
         <param name="Icp/Iterations"          type="string" value="20"/>
-        <param name="Icp/VoxelSize"           type="string" value="0.01"/> <!-- 0.01 works pretty well -->
+        <param name="Icp/VoxelSize"           type="string" value="0.1"/> <!-- 0.01 works pretty well -->
```

Comment:
why is this? Shouldn't increasing voxel size lower the localization accuracy? or did we want to prevent some errors happening due to small voxelSize?

Reply:
Oh that's to prevent all the error messages about leaf size coming up on the terminal. Sure smaller voxels are more accurate but we thought they may be eating up compute at the planner stage. This was one of the experimentation changes. Don't know how it made through to the PR but I think we can keep it as it removes the warning messages on the slam terminal